core/src/main/java/org/apache/carbondata/core/mutate/SegmentUpdateDetails.java

```diff
@@ -84,6 +88,21 @@ public SegmentStatus getSegmentStatus() {
     return this.segmentStatus;
   }
 
+  public Set<String> getDeltaFileStamps() {
+    return deltaFileStamps;
+  }
+
+  public void setDeltaFileStamp(String deltaFileStamp) {
```

Comment:
```suggestion
  public void addDeltaFileStamp(String deltaFileStamp) {
```

It is better to call it `add` since internally it is adding to a set

Reply:
ok